src/Datadog.Trace.ClrProfiler.Managed/Integrations/WebRequestIntegration.cs

```diff
@@ -115,13 +109,7 @@ public static object GetResponse(object webRequest, int opCode, int mdToken, lon
         /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
         /// <returns>Returns the value returned by the inner method call.</returns>
         [InterceptMethod(
-            TargetAssembly = "System", // .NET Framework
-            TargetType = WebRequestTypeName,
-            TargetSignatureTypes = new[] { "System.Threading.Tasks.Task`1<System.Net.WebResponse>" },
-            TargetMinimumVersion = Major4,
-            TargetMaximumVersion = Major4)]
-        [InterceptMethod(
-            TargetAssembly = "System.Net.Requests", // .NET Core
+            TargetAssemblies = new[] { "System", "System.Net.Requests" },
```

Comment:
Should we split these additions into another PR?

Reply:
Sure, I can split that out.

Reply:
Done in 0f6f8dd3bc3ffe32752f6e884b195dc656b2d86e